resource_work_time_from_contracts/models/resource_mixin.py

```diff
@@ -96,7 +96,7 @@ def list_normal_work_time_per_day(self, from_datetime, to_datetime, domain=None)
             day += delta
         return result
 
-    def get_work_days_data(
+    def _get_work_days_data_batch(
```

Comment:
Per the commit message, which indirectly links to https://github.com/odoo/odoo/pull/56477, I think this change (and all other changes in the commit 7bbb859bc545e39a50980867077177e30714badf) should be squashed into the 13.0 migration commit.

Reply:
done